### DISCUSSION TITLE
model-viewer not working after new release.

### DISCUSSION THREAD

**Discussion by faiqah54** [google/model-viewer#3841] [Q&A]
Hi

I'm facing problem after the new release.model viewer worked great before but today its not working and no error show.Can anybody have any idea.

Thanks

**Comment by elalish** [comment]
There's no way we can help with so little information. If you can file an issue and fill in the entire issue template, we'll be happy to help. Also please ensure you test with multiple versions of MV so you can show the difference and we can repro.
